File: JAVA PROGRAMMING/Test18.java
// 18. PRINT EVEN NUMBERS FROM 0 TO N
import java.util.Scanner;
public class Test18 {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);

        System.out.println("Enter Range Value: ");
        int range = in.nextInt();

        for(int i = 0; i <= range; i++) {
            if(i % 2 == 0) {
                System.out.print(i+" ");
            }
        }
        in.close();
    }
}

File: JAVASCRIPT PROGRAMMING/Test18.js
"use strict";
// 18. PRINT EVEN NUMBERS FROM 0 TO N
const func = (range) => {
    for (let i = 0; i <= range; i++) {
        if (i % 2 == 0) {
            process.stdout.write(`${i} `);
        }
    }
    console.log();
};
func(10);
func(20);